packages/release/prepare-local-fs.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Prepare Local Filesystem (/local)
+DefaultDependencies=no
+Wants=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dDATA.device
+After=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dDATA.device
+Before=local.mount
```

Comment:
nit: I would do this in `local.mount` using `After` and `Requires`, to be able to express the "requires" part in the usual way